public/Disable-DbaReplPublishing.ps1

```diff
@@ -0,0 +1,100 @@
+function Disable-DbaReplPublishing {
```

Comment:
Purely because of the method used and referenced this should get named as `Remove-DbaReplPublisher`

Reply:
Consider it from this thought: If I "disable" it, is the "enable" command going to just bring it back up or create it a new object? If enabling creates a new object that is not disable/enable then.

Reply:
Maybe we should use `Register` and `Unregister` instead?